File: smp2017-Python-maulik/cs50/pset6/mario.py
height=-1
while not(height <=23 and height >=0) :
        print('Height :',end='')
        height=int(input())

space=height-1

for i in range(height):
    print(' '*space,end='')
    print('#'*(i+1),end='')
    print('  ',end='')
    print('#'*(i+1))
    i+=1;
    space-=1

File: smp2017-Python-maulik/cs50/pset1/mario.c
#include <stdio.h>

int main(void)
{
	int i,j,ht,space;

	printf("Enter height of double pyramid :");
	scanf("%d",&ht);

	space=ht;

	for(i=0;i<ht;i++,space--)
	{
		for(j=1;j<space;j++)
			printf(" ");
		for(j=0;j<=i;j++)
			printf("#");
		printf("  ");
		for(j=0;j<=i;j++)
			printf("#");
		printf("\n");
	}

	return 0;
}